snuba/web/db_query.py

```diff
@@ -251,7 +254,10 @@ def raw_query(
         if isinstance(cause, RateLimitExceeded):
             stats = update_with_status("rate-limited")
         else:
-            logger.exception("Error running query: %s\n%s", sql, cause)
+            with configure_scope() as scope:
+                if isinstance(cause, ClickhouseError):
+                    scope.fingerprint = ["{{default}}", str(cause.code)]
+                logger.exception("Error running query: %s\n%s", sql, cause)
```

Comment:
I am not sure that either of these belong here, but it seems to make sense to keep them together regardless.